pkg/config/config.go

```diff
@@ -174,7 +175,7 @@ func InitCliConfig(configAndStacksInfo schema.ConfigAndStacksInfo, processStacks
 	configFilePath4 := os.Getenv("ATMOS_CLI_CONFIG_PATH")
 	if len(configFilePath4) > 0 {
 		u.LogTrace(atmosConfig, fmt.Sprintf("Found ENV var ATMOS_CLI_CONFIG_PATH=%s", configFilePath4))
-		configFile4 := filepath.Join(configFilePath4, CliConfigFileName)
+		configFile4 := path.Join(configFilePath4, CliConfigFileName)
```

Comment:
This PR is reverting a change from `path.Join` to → `filepath.Join` due to inadequate testing. We will open a new PR to migrate from `path.Join` to → `filepath.Join`, so that we do not break `main`.